### DISCUSSION TITLE
Suggestion for survey-text

### DISCUSSION THREAD

**Discussion by dcousin3** [jspsych/jsPsych#1177] [Ideas]
I noticed that survey text input boxes allow autocomplete. Hence, participants can see their prior responses in a pull-down list when they start typing in the input box. 

I manually modified survey-text to have 'autocomplete="off"' in the input zone to avoid this behavior. However, that might be an option in the plug-in.

Keep the god work!

**Comment by jodeleeuw** [comment]
Hey @dcousin3, 

We totally agree! This has already been implemented for 6.2 and will be part of that release. (#1063)
